build.sbt

```diff
@@ -843,6 +843,23 @@ lazy val zio1Json = (projectMatrix in file("json/zio1-json"))
   .jsPlatform(scalaVersions = scala2 ++ scala3, settings = commonJsSettings)
   .dependsOn(core, jsonCommon)
 
+lazy val tethysJson = (projectMatrix in file("json/tethys-json"))
+  .settings(
+    name := "tethys-json",
+    libraryDependencies ++= Seq(
+      "com.tethys-json" %% "tethys-core" % "0.26.0",
```

Comment:
maybe define a constant in the top of the build file with the version?

Reply:
There is support, but the new version only supports 3 scala; for 3 scala you need to make a separate implementation.